tests/codegen/debuginfo-proc-macro/auxiliary/macro_def.rs

```diff
@@ -0,0 +1,10 @@
+//@ no-prefer-dynamic
```

Comment:
This *probably* needs `//@ force-host`

Reply:
Thanks, I agree that's probably it. Could I get another try run?

Reply:
Sure.